src/error_utils.mjs

```diff
@@ -19,7 +19,16 @@ function formatAjvErrors(validationErrors) {
         if (error.message === 'must NOT be valid') return;
 
         let formattedError = `${error.instancePath} ${error.message}`;
-        if (error.params.additionalProperty) formattedError += `. Found extra property '${error.params.additionalProperty}'`;
+        if (suffixes) {
+            if (error.params.additionalProperty) {
+                formattedError += `. Found extra suffix '${suffixes[error.params.additionalProperty]}' at index ${error.params.additionalProperty}`;
+            } else if (error.params.allowedValues) {
+                const idx = error.instancePath ? Number(error.instancePath.split('/')[1]) : error.params.additionalProperty;
+                formattedError = `Suffix '${suffixes[idx]}' at index ${idx} ` + formattedError;
```

Comment:
Very minor: I think this prints the index twice? It's already in `formattedError`, so we can likely get rid of `${idx}` in this case

Reply:
I added this because not all permutations of `formattedError` include it, so I'd rather have some duplication then have to guess.

Reply:
Ah that makes sense! Thanks for explaining :)